unix/povconfig/syspovconfigbackend.h

```diff
@@ -50,4 +50,17 @@
 #define POV_USE_DEFAULT_TASK_INITIALIZE 1
 #define POV_USE_DEFAULT_TASK_CLEANUP    1
 
+/// @note
+///     In regression testing an old scene for the 3.7.1 release found
+///     linux machines needed more stack storage than windows.
+///     See too @ref POV_THREAD_STACK_SIZE
+///
```

Comment:
Doxygen may fail to recognize this comment to be associated with `POV_THREAD_STACK_SIZE` if the macro is already set earlier; hence the `/// @def POV_FOO` whenever we provide defaults for a macro in `source/base/configbase.h` and similar places, which explicitly instructs Doxygen that the comment block applies to that macro.

My suggestion would be to make this a plain old C++ comment for now, since we're not generating Doxygen docs from the platform-specific configs at the moment anyway.